### ISSUE TITLE
About performance

### ISSUE THREAD

**Issue by sampaioxsamuel** [TonyPSR/RYZENTOSH-Catalina-MSI-B450#1]
Hey, do you still using this build? if so how good the performance is because I got the same specs and the UI seems kinda slow

**Comment by TonyPSR** [comment]
Yeah. The UI is laggy especially while in YouTube or heavy browser apps. This could be mostly due to GT 710. It's not only old and slow but lacks the ability to decode many modern formats.

I've seen other builds with R7 2700 and RX 580/5700. Those are buttery smooth.

I personally used it for XCODE for a while back. For me it was fine. But not the best experience.
